lib/services/rocket.py

```diff
@@ -0,0 +1,295 @@
+        """
+        Get a rocketpy finset object.
+
+        Returns one of:
+            RocketPyTrapezoidalFins
+            RocketPyEllipticalFins
+        """
+        match kind:
+            case "TRAPEZOIDAL":
+                rocketpy_finset = RocketPyTrapezoidalFins(
+                    n=fins.n,
+                    name=fins.name,
+                    root_chord=fins.root_chord,
+                    tip_chord=fins.tip_chord,
+                    span=fins.span,
+                    cant_angle=fins.cant_angle,
+                    rocket_radius=fins.radius,
+                    airfoil=fins.airfoil,
+                )
+            case "ELLIPTICAL":
+                rocketpy_finset = RocketPyEllipticalFins(
+                    n=fins.n,
+                    name=fins.name,
+                    root_chord=fins.root_chord,
+                    span=fins.span,
+                    cant_angle=fins.cant_angle,
+                    rocket_radius=fins.radius,
+                    airfoil=fins.airfoil,
+                )
+            case _:
+                rocketpy_finset = RocketPyTrapezoidalFins(
+                    n=fins.n,
+                    name=fins.name,
+                    tip_chord=fins.tip_chord,
+                    root_chord=fins.root_chord,
+                    span=fins.span,
+                    cant_angle=fins.cant_angle,
+                    rocket_radius=fins.radius,
+                    airfoil=fins.airfoil,
+                )
+        rocketpy_finset.position = fins.position
+        return rocketpy_finset
+
+    @staticmethod
+    def get_rocketpy_tail(tail: Tail) -> RocketPyTail:
+        """
+        Get a rocketpy tail object.
+
+        Returns:
+            RocketPyTail
+        """
+        rocketpy_tail = RocketPyTail(
+            name=tail.name,
+            top_radius=tail.top_radius,
+            bottom_radius=tail.bottom_radius,
+            length=tail.length,
+            rocket_radius=tail.radius,
+        )
+        rocketpy_tail.position = tail.position
+        return rocketpy_tail
+
+    @staticmethod
+    def get_rocketpy_parachute(parachute: Parachute) -> RocketPyParachute:
+        """
+        Get a rocketpy parachute object.
+
+        Returns:
+            RocketPyParachute
+        """
+        rocketpy_parachute = RocketPyParachute(
+            name=parachute.name,
+            cd_s=parachute.cd_s,
+            trigger=parachute.trigger,
+            sampling_rate=parachute.sampling_rate,
+            lag=parachute.lag,
+            noise=parachute.noise,
+        )
+        return rocketpy_parachute
+
+    @staticmethod
+    def check_parachute_trigger(expression: str) -> bool:
+        """
+        Check if the trigger expression is valid.
+
+        Args:
+            expression: str
+
+        Returns:
+            bool: True if the expression is valid, False otherwise.
+        """
+
+        class InvalidParachuteTrigger(Exception):
+            pass
```

Comment:
Define this class out of this method, possibly in a different file.

Reply:
patched in https://github.com/RocketPy-Team/Infinity-API/pull/25/commits/e323123c9e5faa5989f38917e8c3752f90a54b4e